.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,8 @@ After:
 
 ## Documentation
 - No documentation needed: **add explanation. This can't be used if there is a GUI diff**
-- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)
+- [Documentation issue](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31) was created (GitHub automatic link expected below)
```

Comment:
Is this right?

SUSE/doc-susemanager is public but SUSE/spacewalk is not.

Should this rather be uyuni-project/uyuni-docs ?

Reply:
Docs track issues in Spacewalk, just like the developers.

Reply:
@paususe As @Loquacity wrote: the doc issues are tracked like "development" issues (which we also link in the upstream PRs). I agree that this is a bit unhandy for external contributors, so maybe we could rephrase the instructions a bit:

`(For SUSE Manager contributors): [Documentation issue](url) was created.`

Would it make more sense?

From my own experience I can say that not having such url in the template leads to (even more) forgetting to address the documentation.

Reply:
External contributors can use the "report an issue" link in the Resources menu of the docs, which opens an issue in the Uyuni docs repo. I see them there, and create a duplicate in the Spacewalk repo for us to track. Either way, we have a process to handle them.

Reply:
@hustodemon 

What about something like this?
``
Documentation issue for [Uyuni](https://github.com/uyuni-project/uyuni-docs/issues/new) or [SUSE Manager](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31) was created (GitHub automatic link expected below)
```

Reply:
> External contributors can use the "report an issue" link in the Resources menu...

But the purpose of the link in the template is to suggest the contributors to create a doc issue together with their PR. They don't know, that they need to go to the Resources menu.

> What about something like this? Documentation issue for [Uyuni]...

I think that stating "issue for uyuni OR issue for suma" is a bit confusing: for a SUMA devel, it's not absolutely clear whether they should create suma or uyuni issues. I made this clear in my latest commit: suma devels create suma issues, community devels create uyuni issues.